server/cmt_cmds.go

```diff
@@ -249,3 +258,96 @@ $ %s query block --%s=%s <hash>
 
 	return cmd
 }
+
+func BootstrapStateCmd(appCreator types.AppCreator) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "bootstrap-state",
+		Short: "Bootstrap CometBFT state at an arbitrary block height using a light client",
+		Long: `
+Bootstrap CometBFT state at an arbitrary block height using a light client
+`,
+
+		Args: cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			serverCtx := GetServerContextFromCmd(cmd)
+			logger := log.NewLogger(cmd.OutOrStdout())
+			cfg := serverCtx.Config
+
+			height, err := cmd.Flags().GetInt64("height")
```

Comment:
nit, define a `heightFlag` / `flagHeight` var within `BootstrapStateCmd` to use here and in the flag definition

Reply:
it seems the `height` flag is there automatically?

Reply:
It is defined L346. but that's a minor nit, feel free to ignore.